### PR TITLE
Correct German IA according to #2960

### LINKED ISSUE #2960: German Phrases Cheat Sheet: One small mistranslation/One small addition
Change to "Going out for dinner" section:
- Appetizer
- Vorspeise

Suggested addition to "Basics" section:
- Goodbye!
- Tschüss!

   

------
IA Page:  http://duck.co/ia/view/german_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @AdamSC1-ddg

### PR DESCRIPTION
**Pull request by platschi**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ x] Improvement
    - [x ] Bug fix: **`{IA Name}: Fix #2960`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

- Changed "Mein Name ist.." to the more common "Ich heiße..."
- Added "Goodbye!" Translation
- Changed "Entree" to "Appetizer"

###### Which issues (if any) does this fix?

Fixes #2960 - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/german_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@platschi Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by zekiel** [comment]
@platschi  does that Instant Answer page still exist?

**Comment by tagawa** [comment]
Thank you very much for making these changes @platschi. Unfortunately it looks like there are merge conflicts but GitHub has provided some instructions for fixing them. Please could you have a go and see if it works?

**Step 1:** From your project repository, check out a new branch and test the changes.

`git checkout -b platschi-master master`
`git pull git://github.com/platschi/zeroclickinfo-goodies.git master`

**Step 2:** Merge the changes and update on GitHub.

`git checkout master`
`git merge --no-ff platschi-master`
`git push origin master`

File: share/goodie/cheat_sheets/json/language/german.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
       ],
       "Basics" : [
          {
-            "val" : "Meine Name ist...",
+            "val" : "Ich heiße...",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/language/german.json]: This change has already been implemented in the Instant Answer. Remove this and your branch should have no conflicts 👍

**Comment by moollaza** [comment]
Continued here: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3410

**Comment by tagawa** [comment]
Thank you very much for adding these improvements @platschi - they've now been merged and should be live in a few days.

**Pull request merged**